setup.py

```diff
@@ -30,13 +38,24 @@ def get_version():
 URL = 'http://nilearn.github.com'
 LICENSE = 'new BSD'
 DOWNLOAD_URL = 'http://nilearn.github.com'
-VERSION = get_version()
+VERSION = _VERSION_GLOBALS['__version__']
+
 
 if __name__ == "__main__":
-    old_path = os.getcwd()
+    if is_installing():
+        module_check_fn = _VERSION_GLOBALS['_check_module_dependencies']
+        module_check_fn(preinstalled_modules_only=True)
+
+    # Make sources available using relative paths from this file's directory.
     local_path = os.path.dirname(os.path.abspath(sys.argv[0]))
     os.chdir(local_path)
```

Comment:
I am a bit confused, don't you need the os.chdir to be at module-scope level because of the `open('README.rst)`? I seem to remember that was the case at one point, but maybe I just dreamt about the whole thing.